buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -50,16 +102,34 @@ data class Repository(
  * See `publish.gradle` for details of the publishing process.
  */
 object PublishingRepos {
+
     val mavenTeamDev = Repository(
```

Comment:
I guess we can kill this one, right? We're not using it since `1.0`.

Reply:
I guess so, but @dmdashenkov is current reworking the way we publish. Let's do the clean-up after he finishes.